code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -239,13 +243,16 @@
 	reagent_state = SOLID
 	color = "#FFFFFF" // rgb: 255, 255, 255
 	taste_mult = 1.5 // stop sugar drowning out other flavours
-	nutriment_factor = 10 * REAGENTS_METABOLISM
+	nutriment_factor = 10
 	metabolization_rate = 2 * REAGENTS_METABOLISM
 	overdose_threshold = 200 // Hyperglycaemic shock
 	taste_description = "sweetness"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 	default_container = /obj/item/reagent_containers/condiment/sugar
 
+/datum/reagent/consumable/sugar/synthetic
```

Comment:
Renember to edit the chem dispenser to dispense this.

Reply:
True on the part about affecting chemist gameplay, That said, I foresee this as probably just bringing back the old strat of botany always making a chem dispenser, though the chef might also do it. Plus, the bartender benefits heavily from it and would benefit even more.

Actually now that I realize it, the soda dispenser makes sugar too, that's... already in service.

Reply:
Yeah, I have in my TODO to check whether any other consumables affect chem things. May adjust things for sugar and other things if any.